cypress/integration/ambianic-tests/timeline.spec.js

```diff
@@ -0,0 +1,45 @@
+/// <reference types="cypress" />
+
+context('Timeline', () => {
+    before(() => {
+      cy.visit('http://localhost:8080')
+      cy.get('[data-cy=timeline]').click()
+    })
+
+    it('Should browse to timeline', () => {
+        cy.url().should('include', '/timeline')
+    })
+
+    it('Should render connection card', () => {
+        cy.get('[data-cy=connectioncard]').contains('Connecting to Ambianic Edge device...')
+    })
+
+    /**
+     * This will be removed once we have decided on the flow
+     * for future layout/flow in UI / Edge connection
+     */
+    it('Should browse to edgeconnect', () => {
+
+        cy.wait(500)
+
+        cy.get('[data-cy=settings]').click()
+
+        cy.url().should('include', '/edge-connect')
+    })
+
+    // Try to get this to work once we have mock data
+
+    // it('Get timeline data', () => {
+    //     cy.get('[data-cy=timelinedata]').then(($result) => {
+    //         console.log($result)
+    //     })
+    // })
+
+    // it('Should generate no results in timeline', () => {
+    //     cy.get('[data-cy=timeline]').click()
+
+    //     cy.wait(5000)
+
+    //     cy.get('.infinite-status-prompt').contains('No results :(')
```

Comment:
@BKristenssonAlfsson If I understand correctly your intention here, you commented out tests that verify data in the timeline vue page for now until we add edge in the CI runtime? Is that correct?

Reply:
@ivelin Yes, that is correct. Nasr and me are now working on mock data in Vue, and then I'll head down in the docker file to connect to an Edge and use that for verifications. Less brittle then